src/Pagination.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Choredo;
+
+class Pagination
```

Comment:
This is pedantic, but I feel like `Pagination` is a weird name for a DTO. in the past I've used `OffsetLimit`. Thoughts?